packages/meditrak-server/src/app/createApp.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import { TupaiaDatabase } from '@tupaia/database';
+import { MicroServiceApiBuilder } from '@tupaia/server-boilerplate';
+
+/**
+ * Set up express server with middleware,
+ */
+export function createApp() {
+  const app = new MicroServiceApiBuilder(new TupaiaDatabase()).build();
```

Comment:
Okay so here's the thing. While this is being dubbed an 'orchestration server' due to its role in the architecture, as of now the `meditrak-app` doesn't interact with the server in the same way all our other front-end clients do. 

They all have a simple, browser->server relationship, which requires login/logout routes, session management, etc.

The meditrak-app on the other hand handles most of that stuff locally, and just relies on the server to perform certain operations, and the mechanics of the communication is more similar to our 'microservice' style.

I think for now we'll just use a `MicroServiceApiBuilder` for this, and we'll see if we want to switch to the `OrchestrationApiBuilder` in the future. We may also want to consider renaming, or even consolidating the two ApiBuilders...